CHANGELOG.md

```diff
@@ -1,5 +1,33 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2023-04-02
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/).
+
+### Announcements
+ 
+* Starting on March 21, 2023, traffic to k8s.gcr.io will be redirected to registry.k8s.io, following the [community announcement](https://kubernetes.io/blog/2023/03/10/image-registry-redirect/).
+* Docker container runtime will be retired for Windows nodepools on May 1, 2023. After docker container runtime is retired, you may remain on existing deployed instances but scaling operations will fail, nodepool creation will fail, and you will be out of support. Follow the detailed steps [in our documentation](https://learn.microsoft.com/azure/aks/learn/quick-windows-container-deploy-cli) to upgrade to containerd.
+* AKS will [deprecate](https://learn.microsoft.com/azure/aks/supported-kubernetes-versions?tabs=azure-cli#aks-kubernetes-release-calendar) Kubernetes version 1.23 on April 2, 2023. Please upgrade your AKS clusters to version 1.24 or above.
+
+### Release notes
+
+* Feature
+  * [Terminating a long running operation](https://learn.microsoft.com/azure/aks/manage-abort-operations?tabs=azure-cli) on an AKS cluster is now Generally available.
+  
+* Bug Fix
+  * Fixed an issue that NodeNotReady in Cilium clusters with systemd >=249.  
+
+* Behavior Changes
+  * [L7 proxy for cilium is disabled](https://learn.microsoft.com/en-us/azure/aks/azure-cni-powered-by-cilium#limitations) and not supported for GA
```

Comment:
is this limitation only for Azure CNI Cilium? or does it apply for BYO-CNI + Cilium as well? If it's just one of the two, can we update the part within '[]' to reflect the same.

Reply:
```suggestion
  * [L7 proxy for Azure CNI powered by Cilium is disabled](https://learn.microsoft.com/en-us/azure/aks/azure-cni-powered-by-cilium#limitations) and not supported for GA
```